pkg/api/transform_filter.go

```diff
@@ -18,20 +18,20 @@
 package api
 
 type TransformFilter struct {
-	Rules []TransformFilterRule `yaml:"rules" doc:"list of filter rules, each includes:"`
+	Rules []TransformFilterRule `yaml:"rules" json:"rules" doc:"list of filter rules, each includes:"`
 }
 
 type TransformFilterOperationEnum struct {
-	RemoveField              string `yaml:"remove_field" doc:"removes the field from the entry"`
-	RemoveEntryIfExists      string `yaml:"remove_entry_if_exists" doc:"removes the entry if the field exists"`
-	RemoveEntryIfDoesntExist string `yaml:"remove_entry_if_doesnt_exist" doc:"removes the entry if the field doesnt exist"`
+	RemoveField              string `yaml:"remove_field" json:"remove_field field from the entry"`
+	RemoveEntryIfExists      string `yaml:"remove_entry_if_exists" json:"remove_entry_if_exists" doc:"removes the entry if the field exists"`
+	RemoveEntryIfDoesntExist string `yaml:"remove_entry_if_doesnt_exist" json:"remove_entry_if_doesnt_exist" doc:"removes the entry if the field doesnt exist"`
```

Comment:
so, I didn't change the enums because they are JSON values, not JSON keys. I think it's fine to keep them as they are, no?
so changes are only focusing on keys